### ISSUE TITLE
should divide schema into different files.

### ISSUE THREAD

**Issue by calidion** [prisma/prisma#21581]
## Problem

Currently the schema file is not separable, while we developers always develop with decoupled modules, they don't need a whole schema to be defined and the schema's are changing overtime. It is suggested that the file `schema.prisma` be able to include pieces from other files.

## Suggested solution

For example:

```
# user.model
model User {
  id    Int     @id @default(autoincrement())
  email String  @unique
  name  String?
}
```

```
# db.sqlite
datasource db {
  provider = "sqlite"
  url      = env("DATABASE_URL")
}
```

```
# schema.prisma
generator client {
  provider = "prisma-client-js"
}
include "./db.sqlite"
include "./user.model"

```

**Comment by jkomyno** [comment]
Hi @calidion, this request is already on our radar, but so far we have been focusing our attention to more crucial features and improvements. However, there are a few third-party (i.e., with no official Prisma support) tools to help you in this case, as seen on [StackOverflow](https://stackoverflow.com/questions/71163623/how-to-split-prisma-model-into-separate-file). Hopefully this will unblock you until we support this feature request officially!

Closing this as a duplicate of https://github.com/prisma/prisma/issues/2377.
